### ISSUE TITLE
Allow users to disable figure announcements.

### ISSUE THREAD

**Issue by rkingett** [nvaccess/nvda#16102]
### Is your feature request related to a problem? Please describe.

NVDA, as of 2024, does not give the user the ability to disable "figure" announcements. The user should be able to decide if they would like to hear when entering or exiting "figures"

Right now, this setting does not appear in the settings menus.

### Describe the solution you'd like

A checkbox in the document formatting settings pane to disable, or enable, the announcement of "figure"

### Describe alternatives you've considered

None, as the document formatting settings tree is the most logical place for this.

### Additional context.

I would like to have this work across applications, so including Microsoft Word documents as well or anywhere where the "figure" element is encountered.

**Comment by Adriani90** [comment]
Closing as duplicate of #10826.
